server/views/pages/overview/partials/educationAndTrainingTab/_inPrisonQualificationsCompletedInLast12Months.njk

```diff
@@ -37,7 +37,7 @@
                     <td class="govuk-table__cell">{{ completedCourse.isAccredited | formatIsAccredited }}</td>
                     <td class="govuk-table__cell">{{ completedCourse.prisonName if completedCourse.prisonName else completedCourse.prisonId }}</td>
                     <td class="govuk-table__cell">{{ completedCourse.courseCompletionDate | formatDate('D MMMM YYYY') }}</td>
-                    <td class="govuk-table__cell">{{ completedCourse.grade if completedQualification.grade else 'N/A' }}</td>
```

Comment:
The `if completedQualification.grade else 'N/A'` clause had incorrectly been copied from a previous version of this view for a previous design.
2 things are wrong - firstly the test is on the wrong property name that no longer exists, hence it is always showing `N/A` (`completedQualification.grade` when it should be `completedCourse.grade`)

Secondly, the test itself is now pointless and redundant. Given that this table is about showing completed courses, they will by definition have a completion grade 🤷‍♂️